x-pack/plugins/observability_ai_assistant/public/components/chat/chat_body.tsx

```diff
@@ -161,13 +163,13 @@ export function ChatBody({
     }
   };
 
-  const handleChangeHeight = (editorHeight: number) => {
+  const handleChangeHeight = useCallback((editorHeight: number) => {
```

Comment:
This caused the render to loop which made focus be put onto the PromptEditor over and over.